cookbook/email/dev_environment.rst

```diff
@@ -119,6 +119,58 @@ the replaced address, so you can still see who it would have been sent to.
     These are ``X-Swift-Cc`` and ``X-Swift-Bcc`` for the ``CC`` and ``BCC``
     addresses respectively.
 
+Sending to a Specified Address, but with exceptions
```

Comment:
In the Symfony documentation, the titles follow some special notation. That's why this should be reworded as follows (unless @xabbuh suggest other rewording):

```
Original: Sending to a Specified Address, but with exceptions

New: Sending to a Specified Address but with Exceptions
```